sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/AbstractSynchronousInstrument.java

```diff
@@ -68,14 +68,13 @@ public B bind(Labels labels) {
           // acquire but because we requested a specific value only one will succeed.
           boundLabels.remove(entry.getKey(), entry.getValue());
         }
-        instrumentAccumulator.batch(
-            entry.getKey(), entry.getValue().getAggregator(), !unmappedEntry);
+        instrumentProcessor.batch(entry.getKey(), entry.getValue().getAggregator(), !unmappedEntry);
       }
-      return instrumentAccumulator.completeCollectionCycle();
+      return instrumentProcessor.completeCollectionCycle();
     } finally {
       collectLock.unlock();
     }
   }
 
-  abstract B newBinding(InstrumentAccumulator instrumentAccumulator);
+  abstract B newBinding(InstrumentProcessor instrumentProcessor);
```

Comment:
I was following this rename until I hit this. Why are we passing a processor to create a new binding for an instrument? That doesn't seem like the role of the processor to me. I could think that the binding would be registered with the accumulator, not the processor.

Reply:
Actually I need to pass only an Aggregator you can see that all the Bound constructor just call getAggregator. I decided to do that immediately after https://github.com/open-telemetry/opentelemetry-java/pull/2257.